File: dayOne/java/LevelTwo.java
public class LevelTwo{


    public static int [] mostOccuringNumber(int [] collections){

        int appearance = collections[0];
        int appear = 0;
		int count = 0;
		
		for(;count < collections.length; count++){

			int counter = 0;
			for(int counting = count+1; counting < collections.length; counting++){

				if (collections[counting] == collections[count]){
				counter++;
                }
			}
		    if (counter > appear){
                appear = counter;
                appearance = collections[count];
            }
        }
        return collections;
    }

    public static int [] sort(int [] collections){

        int index = 0;

        for (;index < collections.length; index++){
            int counter = index+1;
            for (;counter < collections.length; counter++){
                if (collections[index] < collections[counter]){
                    int swap = collections[index];
                    collections[index] = collections[counter];
                    collections[counter] = swap;
                }
            }
        }
        return collections;
    }
}

File: dayOne/python/level_two.py
def re_occurring(collections):
    highest_count = 0;
    new_list = []

    for element in collections:
        appearance = 0
        for counter in range(len(collections)):
            if (element == collections[counter]):
                appearance+=1

        if (appearance > highest_count):
            highest_count = appearance
            new_list = [element]
#            new_list.append(collections[count])
        elif (appearance == highest_count) and element not in new_list:
            new_list.append(element)
    return new_list


def sort_list(collections):    
    for index in range(len(collections)):
        for counter in range(index+1, len(collections)):
            if (collections[index] < collections[counter]):
                swap = collections[index]
                collections[index] = collections[counter]
                collections[counter] = swap
    return collections    


